submit_cgap/scripts/submit_ontology.py

```diff
@@ -42,7 +48,22 @@ def main(simulated_args_for_testing=None):
                 validate_only=args.validate_only,
                 app=args.app,
                 submission_protocol=args.submission_protocol,
-        )
+                verbose=args.verbose, debug=args.debug)
+
+
+def verify_ontology_file(ontology_filename: str) -> bool:
+    if not os.path.exists(ontology_filename):
+        show(f"ERROR: Cannot find specified ontology (JSON) file: {ontology_filename}")
+        return False
+    try:
+        with io.open(ontology_filename, "r") as f:
+            ontology_json = json.load(f)
+            ontology_term_count = len(ontology_json["terms"])
+    except Exception:
+        show(f"ERROR: Cannot load specified ontology (JSON) file: {ontology_filename}")
+        return False
```

Comment:
Again untested, but I think:
```suggestion
        raise ScriptFailure(f"ERROR: Cannot load specified ontology (JSON) file: {ontology_filename}")
```